doc/README.md

```diff
@@ -0,0 +1,67 @@
+## Adding GRPC Support to Tailcall Overview
+
+This proposal outlines an approach for adding GRPC support to Tailcall to allow generating GRPC server stubs from GraphQL schemas.
+
+
+# Design Goals
+
+The primary design goals are:
+
+Allow configuring GRPC servers alongside HTTP servers in Tailcall config
+Generate GRPC server stubs from GraphQL schemas
+Enable handling GRPC requests with Tailcall resolvers
```

Comment:
This is interesting, but our goal initially is to only support GRPC for upstream services.

Reply:
By upstream services as in only requiring the GRPC client?

Reply:
Yes, just the GRPC client. The server would expose a unified GraphQL endpoint over HTTP like it does right now.